src/sources/file.rs

```diff
@@ -268,6 +274,10 @@ fn default_line_delimiter() -> String {
     "\n".to_string()
 }
 
+const fn default_rotate_wait_ms() -> Duration {
+    Duration::from_millis(5_000)
```

Comment:
We will need to submit this to upstream and their behavior is indefinite.  The default should not break existing deployments and we should modify the CLO to configure it to our needs